src/server/repository/public/userProvider/index.ts

```diff
@@ -1,9 +1,11 @@
 import { create } from './create'
+import { getUserProviders } from './getUserProviders'
 import { read } from './read'
 import { update } from './update'
 
 export const UserProviderRepository = {
   create,
+  getUserProviders,
```

Comment:
This better to call getMany

Reply:
even if it returns a list of strings and not the whole object?